### PR TITLE
Feat/slider

### PR DESCRIPTION
**Pull request by Wandkleister**

### CODE REVIEW

File: _includes/slider.liquid [hunk 1]
```diff
@@ -1,3 +1,28 @@
-<div class="slider__container">
-	Slider
+<div class="slider">
```
Comment (tam-kis) [inline _includes/slider.liquid:1]: Should this be a `<section>` since it's the slider area?

File: _includes/slider.liquid [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+	<h3>Slider heading</h3>
```
Comment (tam-kis) [inline _includes/slider.liquid]: I would add `<h1>` to the stage and `<h2>` to all the titles of sections

File: styles/includes/_slider.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.glide {
```
Comment (tam-kis) [inline styles/includes/_slider.scss]: Do you plan to add more CSS? Maybe too much nesting

File: package.json [hunk 4]
```diff
@@ -38,5 +38,8 @@
   "bugs": {
     "url": "https://github.com/sinnerschrader/sinnerschrader-reboot/issues"
   },
-  "homepage": "https://github.com/sinnerschrader/sinnerschrader-reboot#readme"
+  "homepage": "https://github.com/sinnerschrader/sinnerschrader-reboot#readme",
+  "dependencies": {
```
Comment (timbru31) [inline package.json:42]: A bit nitpicky and I know npm adds something new at the bottom, but:  
I general I prefer dependencies ordered before devDeps.

File: _includes/js/slider.js [hunk 5]
```diff
@@ -1,10 +1,48 @@
...
+	}
+
+	mountSlider(options) {
+		this.slider = new Glide(".glide", options);
+		this.slider.mount();
+	}
+
+	rotateCircleText() {
+		this.rotationAngle += 60;
+		this.circleText.setAttribute("style", "transform: rotate(" + this.rotationAngle + "deg)");
```
Comment (timbru31) [inline _includes/js/slider.js:45]: Up for discussions - do we prefer template strings?
```suggestion
		this.circleText.setAttribute("style", `transform: rotate(${this.rotationAngle}deg)`);
```

File: _includes/slider.liquid (hunk 1 above)
Comment (Wandkleister) [reply _includes/slider.liquid]: Agree ✅

File: _includes/slider.liquid (hunk 2 above)
Comment (Wandkleister) [reply _includes/slider.liquid]: Updated ✅

File: styles/includes/_slider.scss (hunk 3 above)
Comment (Wandkleister) [reply styles/includes/_slider.scss]: Removed unnecessary nesting ✅

File: package.json (hunk 4 above)
Comment (Wandkleister) [reply package.json]: Understandable and changed ✅

File: _includes/js/slider.js (hunk 5 above)
Comment (Wandkleister) [reply _includes/js/slider.js]: Yep template strings is better - need to get rid of that old habit ^^
Changed ✅

**Review by tam-kis — APPROVED** [review]

**Pull request merged**